README.md

```diff
@@ -50,62 +50,95 @@ pip install -e .
 
 Make sure that you can run `python -c "import gtsfm; import gtsam; print('hello world')"` in python, and you are good to go!
 
+## Try It on Google Colab  
+
+For a quick hands-on example, check out this Colab notebook [![Colab notebook](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1H375PLnh-w5XhUOjrGoSAmczjqGYggD8?usp=sharing#scrollTo=2N_uauEtNlUP)
+
+
 ## Usage Guide (Running 3D Reconstruction)
 
-Before running reconstruction, if you intend to use modules with pre-trained weights, such as SuperPoint, SuperGlue, or PatchmatchNet, please first run:
+Before running reconstruction, if you intend to use modules with pre-trained weights (e.g., **SuperPoint, SuperGlue, or PatchmatchNet**), first download the model weights by running:  
 
 ```bash
 ./download_model_weights.sh
-```
+```  
+
+### Running SfM  
 
-To run SfM with a dataset with only an image directory and EXIF, with image file names ending with "jpg", please create the following file structure like
+To process a dataset containing only an **image directory and EXIF metadata**, ensure your dataset follows this structure:  
```

Comment:
the EXIF metadata is optional, but only the 1dsfm loader supports that. 

Maybe we can add a line that says something like: 
If you are using a dataset without EXIF, use the one_d_sfm_loader: 

<example_command>

?